components/raftstore/src/store/entry_storage.rs

```diff
@@ -399,6 +400,130 @@ impl Drop for EntryCache {
     }
 }
 
+/// A cache that maintains term information for recently updated Raft log
+/// entries.
+///
+/// This cache optimizes term lookups by storing term-to-index mappings for
+/// recent entries. Each cache entry contains:
+/// - term: The term number of a sequence of entries
+/// - start_index: The start raft log index that shares this term
+///
+/// The cache maintains entries in monotonically increasing order by term,
+/// allowing quick lookups of terms for any given log index. The small fixed
+/// capacity helps keep memory overhead minimal while caching the most relevant
+/// term information.
+struct TermCache {
+    cache: VecDeque<(u64, u64)>, // (term, start_index)
```

Comment:
Do you think it is worthy using VecDeque<(start_idx, end_idx, term)>, so that the size will be smaller, and search can be faster

Reply:
oh, i didn't notice the queue size is only 8